cmd/gaia/init/genesis_accts_test.go

```diff
@@ -56,8 +56,8 @@ func TestAddGenesisAccount(t *testing.T) {
 			args{
 				app.GenesisState{},
 				addr1,
-				sdk.Coins{sdk.NewInt64Coin("stake", 50)},
-				sdk.Coins{sdk.NewInt64Coin("stake", 100)},
+				sdk.NewCoins(sdk.NewInt64Coin("stake", 50)),
```

Comment:
Maybe we parameterize this somewhere or use the `staking.DefaultBondDenom`?

Reply:
There are a number of these in tests below as well. This isn't critical, but is best practice.

Reply:
makes sense don't think it needs to happen in this PR

Reply:
BTW coin denomination here does not matter much here, it's just a test for the `add-genesis-account` command.